storage/queued_country.hpp

```diff
@@ -6,6 +6,9 @@
 namespace storage
 {
 /// Country queued for downloading.
+/// @TODO(bykoianko) This class assume that a map may consist of one or two mwm files.
+/// But single mwm files are used now. So this class should be redisigned to support
```

Comment:
This class (queued_country) is responsible for downloading maps. As far as I see it's a queue for mwms which will be downloaded. We're not planning to support downloading two components mwms in the new downloader. This ability is not supported in the new downloader programming interface and in Storage. Yes, we do need to support the ability to work with old (two component mwms) in our application but not download them. If so why do we need to support two component mwms in queued_country?